packages/cli-platform-android/native_modules.gradle

```diff
@@ -1,588 +0,0 @@
-import groovy.json.JsonSlurper
```

Comment:
I was thinking instead of deleting this file we can just let it print a huge warning on what to do instead

Reply:
Oh that's very reasonable, will update